htdocs/update.bml.text

```diff
@@ -147,5 +147,9 @@
 
 .extradata.sec.custom=The entry was posted with custom access.
 
-.extradata.subject=The entry was posted with the following subject: [[subject]]
+.extradata.subj=The entry was posted with the following subject: 
+
+.extradata.subject.no_subject=(no subject)
+
+.extradata.subject.subject=[[subject]]
```

Comment:
I see how you ended up here by refactoring the existing string, but: there is no reason to use `.extradata.subject.subject` as a translation string if its only purpose is to reprint the value of the variable. You should just print the value of the variable instead of making a translation call.